wg-charter.html

```diff
@@ -0,0 +1,497 @@
+
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process Document</a>:</p>
+
+        <div>
+          <h3 id="w3c-coordination">W3C Groups</h3>
+          <dl>
+            <dt><a href="https://www.w3.org/community/gpu/">GPU for the Web Community Group</a></dt>
+            <dd>The GPU for the Web Community Group developed the specifications adopted by this Working Group. It is expected that the Community Group will continue to drive the technical work on the specifications and incubate new features. This Working Group will work with the GPU for the Web Community Group on shaping the specifications for the Recommendation track.</dd>
+
+            <dt><a href="https://www.w3.org/immersive-web/">Immersive Web Working Group</a></dt>
+            <dd>The Immersive Web Working Group produces specifications to interact with Virtual Reality (VR) and Augmented Reality (AR) devices and sensors, and may add a 3D rendering layer based on the WebGPU API.</dd>
+        <dt><a href="https://www.w3.org/2019/webapps/">Web Applications Working Group</a></dt>
+        <dd>The Web Applications Working Group (WebApps WG) produces specifications that facilitate the development of client-side web applications.</dd>
+
+        <dt><a href="https://www.w3.org/2011/webappsec/">Web Application Security Working Group</a></dt>
+        <dd>This Working Group develops security and policy mechanisms to improve the security of Web Applications, and enable secure cross-site communication.</dd>
+
+        <dt><a href="https://www.w3.org/wasm/">Web Assembly Working Group</a></dt>
+        <dd>This Working Group develops a size- and load-time-efficient format and execution environment, allowing compilation to the web with consistent behavior across a variety of implementations.</dd>
+
+        <dt><a href="https://www.w3.org/groups/wg/webmachinelearning/">Web Machine Learning Working Group</a></dt>
+        <dd>This Working Group develops a dedicated low-level Web API for enabling efficient machine learning inference in the browser.</dd>
+          </dl>
+          <h3 id="external-coordination">External Organizations</h3>
+          <dl>
+          <dt><a title="WHATWG" href="https://whatwg.org/" id="whatwg">Web Hypertext Application Technology Working Group (WHATWG)</a></dt>
+          <dd>The Web Hypertext Application Technology Working Group (WHATWG) is a community of people interested in evolving the web through standards and tests.</dd>
+          <dt><a title="Khronos" href="https://khronos.org/" id="khronos">Khronos</a></dt>
+          <dd>The Khronos Organization is a standards body that develops many graphics-related technologies, including OpenGL, Vulkan and SPIR-V.</dd>
+          </dl>
+        </div>
```

Comment:
@tidoust , I had a little suggestion. Would it be possible to include UTC here, as we have got in touch during the identifier discussion and there is quite large citation of their specs in WGSL? Suggested addition ( for the update of description of normativeness (not this part of WGSL in this document) , I think dneto0 and alan-baker will make better suggestion ):

Unicode Technical Committee (UTC)

Unicode Technical Group is part of the Unicode Consortium that maintains and develops the Unicode Standard, including the databases. This WG will review their publications relevant to WebGPU and WGSL standards in the web ecosystem compatibility.

(I based this text on https://www.w3.org/International/groups/wg/charter.html )

Reply:
I added an entry for the UTC, rewording the second sentence as I suspect this group is more interested in exchanging with the UTC than in offering reviews of their publications.

Internationalization being one of the mandatory horizontal reviews, an indirect link with the UTC already exists through the internationalization group, which may suggest a different approach or wording when they review the charter.

Reply:
@tidoust The rewording reads great to me, thank you for properly wording the context!